app/controllers/idv/in_person/ready_to_verify_controller.rb

```diff
@@ -12,12 +12,20 @@ class ReadyToVerifyController < ApplicationController
       before_action :confirm_in_person_session
 
       def show
-        analytics.idv_in_person_ready_to_verify_visit
+        analytics.idv_in_person_ready_to_verify_visit(**opt_in_analytics_properties)
         @presenter = ReadyToVerifyPresenter.new(enrollment: enrollment)
       end
 
       private
 
+      def opt_in_analytics_properties
+        extra = {}
+        if IdentityConfig.store.in_person_proofing_opt_in_enabled
+          extra = { opted_in_to_in_person_proofing: idv_session.opted_in_to_in_person_proofing }
+        end
+        extra
```

Comment:
same comment about removing the assignment
```suggestion
        if IdentityConfig.store.in_person_proofing_opt_in_enabled
          { opted_in_to_in_person_proofing: idv_session.opted_in_to_in_person_proofing }
        else
          {}
        end
```